docs/agents/scpi_psu.rst

```diff
@@ -0,0 +1,95 @@
+.. highlight:: rst
+
+.. _scpi_psu:
+
+==================
+SCPI PSU Agent
+==================
+
+This agent uses Standard Commands for Programmable Instruments (SCPI)
+It works for many power supplies, including the Keithley 2230G 
+and BK Precision 9130. It connects to the PSU over ethernet, and allows
+users to set current, voltage, and turn channels on/off. It also allows for 
+live monitoring of the PSU output. 
+
+.. argparse::
+    :filename: ../agents/scpi_psu/scpi_psu_agent.py
+    :func: make_parser
+    :prog: python3 scpi_psu_agent.py
+
+
+Configuration File Examples
+---------------------------
+Below are configuration examples for the ocs config file and for running the
+Agent in a docker container.
+
+ocs-config
+``````````
+To configure the SCPI PSU Agent we need to add a block to our ocs 
+configuration file. Here is an example configuration block using all of 
+the available arguments::
+
+      {'agent-class': 'ScpiPsuAgent',
+        'instance-id': 'psuK',
+        'arguments': [
+          ['--ip-address', '10.10.10.5'],
+          ['--gpib-slot', '1']
+          ]},
+
+Most power supplies (including the Keithly 2230G and BK Precision 9130) 
+have GPIB ports rather than ethernet ports. Therefore a GPIB-to-ethernet
+converter is required, and the gpib slot must be specified in the ocs
+configuration file. The IP address is then associated with the converter.
+
+Docker
+``````
+The SCPI PSU Agent should be configured to run in a Docker container.
+An example docker-compose service configuration is shown here::
+
+  ocs-psuK:
+    image: simonsobs/ocs-scpi-psu-agent:latest
+    hostname: manny-docker
```

Comment:
Can we change the SAT1 specific `manny-docker` to `ocs-docker`? This is consistent with other examples.